README.rst

```diff
@@ -3,216 +3,62 @@ Karapace
 
 ``karapace`` Your Kafka essentials in one tool
 
+Overview
+========
+
+Karapace supports the storing of schemas in a central repository, which clients can access to
+serialize and deserialize messages. The schemas also maintain their own version histories and can be
+checked for compatibility between their different respective versions.
+
+Karapace rest provides a RESTful interface to your Kafka cluster, allowing you to perform tasks such
+as producing and consuming messages and perform administrative cluster work, all the while using the
+language of the WEB.
+
+Karapace is compatible with Schema Registry 6.1.1 on API level.
 
 Features
 ========
 
-* Schema Registry and Rest Proxy aim to be API compatible with the pre-existing proprietary
-  Confluent Schema Registry and Kafka Rest Proxy
 * Drop in replacement both on pre-existing Schema Registry / Kafka Rest Proxy client and
   server-sides
 * Moderate memory consumption
 * Asynchronous architecture based on aiohttp
+* Supports Avro and JSON Schema. Protobuf development is tracked with `Issue 67`_.
 
-
-Overview
-========
-
-Karapace supports the storing of schemas in a central repository, which
-clients can access to serialize and deserialize messages.  The schemas also
-maintain their own version histories and can be checked for compatibility
-between their different respective versions.
-
-Karapace rest provides a RESTful interface to your Kafka cluster, allowing you to perform
-tasks such as producing and consuming messages and perform administrative cluster work,
-all the while using the language of the WEB.
-
-Karapace is compatible with Schema Registry 6.1.1 on API level with the following exceptions
-
-* Protobuf schemas are unsupported
+.. _Issue 67: https://github.com/aiven/karapace/issues/67
 
 Setup
 =====
 
-Karapace is a Python project, and requires Kafka for its backend storage. There is also a `Docker setup for development`_.
-
-Requirements
+Using Docker
 ------------
 
-Karapace requires Python 3.6 or later and some additional components in
```

Comment:
I removed all of these and just added an instruction to run `pip install -r requirements-dev.txt`